tasks/publish.yaml

```diff
@@ -0,0 +1,38 @@
+includes:
+  - dependencies: ./dependencies.yaml
+  - create: https://raw.githubusercontent.com/defenseunicorns/uds-common/v0.3.3/tasks/create.yaml
+  - publish: https://raw.githubusercontent.com/defenseunicorns/uds-common/v0.3.3/tasks/publish.yaml
+
+tasks:
+  - name: test-bundle
+    description: Build and publish the test bundles
+    actions:
+      - description: Create the ARM64 deps package
+        task: dependencies:create
+        with:
+          options: -a amd64
+
+      - description: Create the ARM64 deps package
+        task: dependencies:create
+        with:
+          options: -a arm64
```

Comment:
```suggestion
          architecture: arm64
```

Reply:
you changed the pattern we were using between the time i wrote this and the time you reviewed it 🤣